### DISCUSSION TITLE
running demo sought

### DISCUSSION THREAD

**Discussion by malcook** [C3BI-pasteur-fr/UTechSCB-SCHNAPPs#5] [Q&A]
is there a version running with some demo data that can be used to check out the features?

**Comment by baj12** [comment]
somehow I don't get notifications for comments. Sorry for not closely monitoring...
there is a directory data with scEx.RData that can be loaded to check out the features.

**Comment by baj12** [comment]
and check out the youtube videos:
https://youtube.com/playlist?list=PL__mZTR-JcdOpa8Llz2Dfcu_mCk8iBCpC
